ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -57,6 +59,48 @@ objects:
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
+    initContainers:
+    - name: share-installer-src
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:src
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+        mkdir /tmp/installer-src
```

Comment:
nit: we should be able to drop this.  `cp` can create the target directory on its own:

```console
$ mkdir -p a/b
$ cd a
$ cp -vR . /tmp/installer-src
‘.’ -> ‘/tmp/installer-src’
‘./b’ -> ‘/tmp/installer-src/b’
```

Once we drop the `mkdir`, the only thing left is the `cp` call.  So we should be able to drop the shell entirely, and go straight to `cp`:

```yaml
command:
- cp
- -R
- .
- /tmp/installer-src
```

or something like that.